h5py_wrapper/lib.py

```diff
@@ -10,25 +10,37 @@
 import tarfile
 
 
-def get_previous_version(version):
+def get_previous_version(version, path):
     """
     Retrieves the given version of the wrapper from github as a tar
     archive and extracts its contents to the current directory.
 
+    Parameters
+    ----------
+    version : str
+        Version number in format 'X.X.X'
+        Note that for version 0.0.1, the it has
+        to be specified as 'v0.0.1'.
+    path : str
+        Path to store the files.
     """
-    base_url = "https://github.com/INM-6/h5py_wrapper/archive/v"
+    base_url = "https://github.com/INM-6/h5py_wrapper/archive/"
     r = requests.get(''.join((base_url, version, ".tar.gz")))
+    # Convert path to str
+    path = str(path)
```

Comment:
why do we need this?

Reply:
Because before, `path` is a `LocalPath` object created by the mechanism of `py.test` for temporary directories. This object cannot be handled by `os.path.join` which only accepts `str` objects. There is a way to join `LocalPath` objects but I think handling the path as string is more general and easier here.

Reply:
OK. Could you clarify that in the comment? `convert LocalPath object to str` or something like that?